test/gurka/test_landmarks.cc

```diff
@@ -327,55 +334,61 @@ TEST(LandmarkTest, TestTileStoreLandmarks) {
   GraphReader gr(landmark_map.config.get_child("mjolnir"));
   auto tile = gr.GetGraphTile(tile_id);
 
+  auto check_landmark = [&](const auto& landmark, const auto& point) {
```

Comment:
both of the checks below did the same thing so i pulled it out into a lambda and reused it